packages/app/lib/src/app/wrapper_page.dart

```diff
@@ -19,6 +20,8 @@ import '../features/home/domain/post_entity.dart';
 import '../utils/hooks.dart';
 import 'router.gr.dart';
 
+final uploadedImageProvider = StateProvider<PlatformFile?>((ref) => null);
```

Comment:
StateProvider is deprecated. Use Provider for immutable state, and NotifierProvider mutable state.

Reply:
Also, don't define providers in pages.